modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartJaguarClientCodegen.java

```diff
@@ -43,11 +45,15 @@
         modelToIgnore.add("file");
     }
 
+    private boolean nullableFields = true;
+
     public DartJaguarClientCodegen() {
         super();
         browserClient = false;
         outputFolder = "generated-code/dart-jaguar";
         embeddedTemplateDir = templateDir = "dart-jaguar";
+
+        cliOptions.add(new CliOption(NULLABLE_FIELDS, "Is the null fields should be in the JSON payload"));
```

Comment:
@jaumard instead of adding another option, what about leveraging the `nullable` attribute (OAS v3) instead?

(in OAS v2, `x-nullable` is used instead)

Reply:
It's not the same ;) my nullableField is for dart generation, it's a global flag for the Json serializer